src/layout-grid.scss

```diff
@@ -73,6 +73,12 @@ $cols: 12;
   min-width: 100%;
   max-width: 100%;
 
+  &--full {
+    min-width: ( ( 1 / 12 ) *100%);
+    max-width: none;
+    flex: 1;
+  }
```

Comment:
You have the same block of code repeated 4 times. Move it to a mixin.

Reply:
Fixed!